templates/acc_active_email.html

```diff
@@ -0,0 +1,7 @@
+{% autoescape off %}  
+Hi {{ name }},  
+Welcome to VOIDOC!
+Please click on the link to confirm your registration,
+http://{{ domain }}{% url 'activate' uid=uid token=token %}
```

Comment:
there should be a way to hide this url from the email body. if yo add a button to access the link or a simple hyper text will improve user experience

Reply:
Done!